src/authentication.ts

```diff
@@ -120,7 +119,7 @@ export function redirectToLogout(
     token_type_hint: refresh_token ? 'refresh_token' : 'access_token',
     client_id: config.clientId,
     post_logout_redirect_uri: config.logoutRedirect ?? config.redirectUri,
-    ui_locales: window.navigator.languages.reduce((a: string, b: string) => a + ' ' + b),
+    ui_locales: window.navigator.languages.reduce((a: string, b: string) => `${a} ${b}`),
```

Comment:
Never seen a `.reduce` like this 😅  Why not just do this?
```suggestion
    ui_locales: window.navigator.languages.join(' '),
```

Reply:
Yeah.... no idea why I did that